lib/spack/spack/test/cmd/audit.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.main import SpackCommand
+
+
+audit = SpackCommand('audit')
+
+
+def test_audit_packages(mutable_config, mock_packages):
+    """Sanity check ``spack audit packages`` to make sure it works."""
+    audit('packages', fail_on_error=False)
+    # Some mock packages have issues by design, to check what would
+    # be Spack behavior in those cases. Ensure that the command returns
+    # non-zero exit code
+    assert audit.returncode == 1
```

Comment:
As is, these tests are a bit "broad", I think it would be sensible to audit specific packages and expect a failure in some cases and a success in others.

Reply:
Should be addressed by 29551c2